languages/javascript/config.json

```
@@ -70,6 +70,12 @@
         "uuid": "39549583-1889-490a-bf98-ffb2e0aefe44",
         "concepts": ["strings"],
         "prerequisites": ["basics"]
+      },
+      {
+        "slug": "closures",
+        "uuid": "5aa39e89-c601-4a66-ab72-5d8512d69e02",
+        "concepts": ["closures"],
+        "prerequisites": ["basics", "arrays", "booleans"]
```

When is are more than 1 prerequisites, the `basics` concept can be omitted:

```suggestion
        "prerequisites": ["arrays", "booleans"]
```